cluster/rbac/manager.go

```diff
@@ -148,17 +149,42 @@ func (m *Manager) UpsertRolesPermissions(c *cmd.ApplyRequest) error {
 
 			// create new permissions
 			for idx := range policies {
-				if req.Roles[roleName][idx].Domain != authorization.SchemaDomain {
-					continue
+				if req.Roles[roleName][idx].Domain == authorization.SchemaDomain {
+					parts := strings.Split(req.Roles[roleName][idx].Resource, "/")
+					if len(parts) < 3 {
+						// shall never happens
+						return fmt.Errorf("invalid schema path")
+					}
+					req.Roles[roleName][idx].Resource = authorization.CollectionsMetadata(parts[2])[0]
```

Comment:
why these changes?

Reply:
before we were filtering for `SchemaDomain` only, now we include `RolesDomain` 
logic didn't change just 
instead of 
`if req.Roles[roleName][idx].Domain != authorization.SchemaDomain {`
we do 
`if req.Roles[roleName][idx].Domain == authorization.SchemaDomain {`

and move the logic inside